src/store/modules/cosmos/bank/index.js

```diff
@@ -0,0 +1,16 @@
+import bank from './bank.js'
+
+export default function init(store) {
+	if (!store.hasModule('modules/cosmos')) {
+		store.registerModule(['modules', 'cosmos'], { namespaced: true })
+	}
```

Comment:
To allow for proper nesting of auto-generated modules we need to register dummy/empty parent module-names. Since we don't know the specific child modules that will be used or  the order they will be registered in, that additional registration is wrapped in an existence conditional.